MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -54,11 +56,13 @@ public class EncodingHelper
         public EncodingHelper(
             IApplicationPaths appPaths,
             IMediaEncoder mediaEncoder,
-            ISubtitleEncoder subtitleEncoder)
+            ISubtitleEncoder subtitleEncoder,
+            IConfiguration config)
         {
             _appPaths = appPaths;
             _mediaEncoder = mediaEncoder;
             _subtitleEncoder = subtitleEncoder;
+            _ffmpegAnalyzeDuration = config.GetValue<string>(Controller.Extensions.ConfigurationExtensions.FfmpegAnalyzeDurationKey) ?? string.Empty;
```

Comment:
Won't this require restarting Jellyfin every time you change the value?

Reply:
Yes, change env requires a restart to take effect.
But our default value 200M is more than enough.

https://github.com/jellyfin/jellyfin/blob/990e4e8e58e2d15f9bbd010e06915a40c9180235/Emby.Server.Implementations/ConfigurationOptions.cs#L19

Reply:
But by using IConfiguration to retrieve the value every time we could in theory avoid it

Reply:
I have changed it to get from GetFFmpegAnalyzeDuration(). https://github.com/jellyfin/jellyfin/blob/4fa7f49ffc6cec2a5be3cf9e5ff0fb727815e6b1/MediaBrowser.Controller/Extensions/ConfigurationExtensions.cs#L79-L85